File: Barber360/lib/models/person.dart
class Person {
  final int id;
  final String name;
  final String email;
  final String identification;
  final String birthDate;

  Person({
    required this.id,
    required this.name,
    required this.email,
    required this.identification,
    required this.birthDate,
  });

  factory Person.fromJson(Map<String, dynamic> json) {
    return Person(
      id: json['Id'],
      name: json['Name'],
      email: json['Email'],
      identification: json['Identification'],
      birthDate: json['BirthDate'],
    );
  }
}

File: Barber_360/Domain/Entidades/Person/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entidades.Person
{
    public class Person
    {
        public int Id { get; set; }
        public string Identification { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public DateTime BirthDate { get; set; }
    }
}
